magpie-persist/src/main/java/io/openraven/magpie/plugins/persist/PersistPlugin.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2021 Open Raven Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.openraven.magpie.plugins.persist;
+
+import io.openraven.magpie.api.MagpieEnvelope;
+import io.openraven.magpie.api.TerminalPlugin;
+import org.slf4j.Logger;
+
+import java.time.Instant;
+
+public class PersistPlugin implements TerminalPlugin<PersistConfig> {
+
+  private final Object SYNC = new Object();
+
+  private static final String ID = "magpie.persist";
+
+  private Logger logger;
+
+  AWSResourceRepo awsResourceRepo;
+
+  @Override
+  public void accept(MagpieEnvelope env) {
+    synchronized (SYNC) {
+      var contents = env.getContents();
+      String tableName = contents.get("resourceType")
+        .asText()
+        .replace(":", "")
+        .toLowerCase();
+
+      if(!awsResourceRepo.doesTableExist(tableName)) {
+        awsResourceRepo.createTable(tableName);
+      }
+
+      awsResourceRepo.upsert(tableName,
+        contents.get("documentId").asText(),
+        contents.get("arn").asText(),
+        contents.get("resourceName").asText(),
+        contents.get("resourceId").asText(),
+        contents.get("resourceType").asText(),
+        contents.get("awsRegion").asText(),
+        contents.get("createdIso").isNull() ? Instant.now().toString() : contents.get("createdIso").asText(),
+        contents.get("updatedIso").isNull() ? Instant.now().toString() : contents.get("updatedIso").asText(),
```

Comment:
Everything is ready to review except for inserting null TIMESTAMPZ here.  I am trying to figure out the way to insert null value